sklearn/manifold/_t_sne.py

```diff
@@ -617,10 +618,22 @@ class TSNE(ClassNamePrefixFeaturesOutMixin, TransformerMixin, BaseEstimator):
         .. versionchanged:: 1.2
            The default value changed to `"auto"`.
 
-    n_iter : int, default=1000
+    max_iter : int, default=None
```

Comment:
I'm not sure what the correct thing to do is here. From looking at other deprecations it seems that the docstring mentions the effective default value (I looked at `make_sparse_spd_matrix` in `sklearn/datasets/_samples_generator.py`), so like this:
```suggestion
    max_iter : int, default=1000
```

Reply:
Ah yes that is much better. I was following #25697 - I think it may be worth updating the docstring of `BayesianRidge` and `ARDRegression` even.